test/functional/feature_config_args.py

```diff
@@ -45,11 +45,11 @@ def test_config_file_parser(self):
                 conf.write("wallet=foo\n")
             self.nodes[0].assert_start_raises_init_error(expected_msg=f'Error: Config setting for -wallet only applied on {self.chain} network when in [{self.chain}] section.')
 
-        main_conf_file_path = os.path.join(self.options.tmpdir, 'node0', 'bitcoin_main.conf')
-        util.write_config(main_conf_file_path, n=0, chain='', extra_config=f'includeconf={inc_conf_file_path}\n')
-        with open(inc_conf_file_path, 'w', encoding='utf-8') as conf:
-            conf.write('acceptnonstdtxn=1\n')
-        self.nodes[0].assert_start_raises_init_error(extra_args=[f"-conf={main_conf_file_path}"], expected_msg='Error: acceptnonstdtxn is not currently supported for main chain')
+        #main_conf_file_path = os.path.join(self.options.tmpdir, 'node0', 'bitcoin_main.conf')
```

Comment:
What's the purpose of keeping this commented instead of deleting it?

Reply:
No particular purpose; any reason to delete it rather than just comment it?